core/find_apk.sh

```diff
@@ -14,7 +14,7 @@ getlatestapk() {
   # decompress apks
   # some apks are lz compressed to work around github file-size limits.
   for foundapklz in $(find "$1" -iname '*.apk.lz'); do
-    lunzip --keep "$foundapklz"
+    lunzip --keep "$foundapklz" &> /dev/null || lzip --decompress --keep "$foundapklz"
```

Comment:
What about testing if lunzip exists (if [[ -e "$(command -v lunzip)" ]]) ? instead of sending the result to /dev/null